ISSUE.md

```diff
@@ -0,0 +1,56 @@
+# Issue reporting on Podman
+
+The Podman team cares for our users and communities.  When someone has a problem with
+Podman and takes the time to report an [issue](https://github.com/containers/podman/issues)
+on our Github, we deeply appreciate the effort.  We want to help. Consider reading
+our [support](SUPPORT.md) document prior to submitting an issue as well.
+
+## Considerations when reporting an issue upstream
+
+### Where to report your issue
+
+If you are running Podman acquired from a Linux distribution and that Linux distribution has a
+bug reporting mechanism, then please report the bug there.
+
+### Development or latest version
+
+We view this Github repository as an upstream repository for development
+and the latest Podman version.
+
+The term "latest version" refers to our mainline development tree or the
+[latest release](https://github.com/containers/podman/releases/latest).
+
+You are welcome to replicate the bug with upstream code and report that.
+
+### Bugs vs features
+
+A bug is when something in Podman is not working as it should or has been described.  A
+feature or enhancement is when you would like Podman to behave differently.
+
+![new issue](issue-feature.png)
+
+### Use the issue template when reporting bugs
+When you report an issue on the upstream repository, be sure to fill out the entire template.
+You must provide the required `podman info` wherever possible as it helps us diagnose
+your report.  If possible, always provide a _reliable reproducer_.  This is extremely
+helpful for us during triage and bug fixing.
+
+### Look through existing issues before reporting a new issue
+Managing issues is a time consuming processes for maintainers.  You can save us time by
+making sure the issue you want to report has not already been reported.  It is appropriate
+to comment on the existing issue with relevant information.
```

Comment:
Add to this: Commenting on 1 year old, closed issues is not helpful. If the issue is closed, please open a new issues with your own details, as the bug you are experiencing may be new.